### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1195] [Galaxy Trivia] [answered]
**ENITNETR**
**TMNCAPLOOOSI**
**ROEASWTF**
**ERVPEDEOL**

**Comment by peterbaccaro** [comment, marked as answer]
- INTERNET
- COSMOPOLITAN
- SOFTWARE
- DEVELOPER

The odd one out is **COSMOPOLITAN** as the other words are associated with information technology.
